tests/Controller/AjaxTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class AjaxTest extends WebTestCase
+{
+    public function testHomepage()
+    {
+        $client = static::createClient();
+
+        $client->request('GET', '/');
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
+        $this->assertResponseHeaderSame('Content-Type', 'application/json');
```

Comment:
Since testing is all about the reporting and these are unit tests , I would recommend splitting this up into two tests so we can get separate reporting on the status code and the content type.